src/extension.ts

```diff
@@ -9,34 +9,34 @@
 *                                                                                 *
 */
 
+// tslint:disable-next-line: no-duplicate-imports
```

Comment:
Not sure if there is an issue to capture this as part of the refactoring, but I do believe that we need to reduce/eliminate the use of `tslint-disable`.
Unless we think we need to update the `tslint-config` (which I don't mind either way)

I know it's not even needed here since the references to the interfaces/types were properly addressed 👍 

Again, No changes requested!